moveit_core/constraint_samplers/test/pr2_arm_kinematics_plugin.cpp

```diff
@@ -128,14 +135,14 @@ int PR2ArmIKSolver::CartToJnt(const KDL::JntArray& q_init, const KDL::Frame& p_i
   {
     if (verbose)
     {
-      RCLCPP_WARN(LOGGER, "Solution : %d", static_cast<int>(solution_ik.size()));
+      RCLCPP_WARN(getLogger(), "Solution : %d", static_cast<int>(solution_ik.size()));
 
       for (int j = 0; j < static_cast<int>(solution_ik[i].size()); ++j)
       {
-        RCLCPP_WARN(LOGGER, "%d: %f", j, solution_ik[i][j]);
+        RCLCPP_WARN(getLogger(), "%d: %f", j, solution_ik[i][j]);
       }
-      RCLCPP_WARN(LOGGER, " ");
-      RCLCPP_WARN(LOGGER, " ");
+      RCLCPP_WARN(getLogger(), " ");
+      RCLCPP_WARN(getLogger(), " ");
```

Comment:
Why do we print two empty strings here? Maybe consider simplifying this

Reply:
good catch, just deleteing these lines